static/styles/portico/portico-signin.scss

```diff
@@ -677,6 +677,13 @@ button.login-social-button {
     }
 }
 
+button#login_auth_button_apple {
+    width: 328px;
+    height: 46px;
+    background-size: auto 100%;
+    background-position: 0px 100%;
+}
```

Comment:
Ah, this reminds me - we will also want the same for the Sign up button, with the button rendering adjustment in `accounts_home.html` (analogical to `login.html`)

Reply:
Oh, I missed it. Apple has only two buttons. One saying "Continue with Apple" and other "Sign in with Apple". set both login and register with "Sign in with apple".

Reply:
For the "sign *up*" case, let's use the "Continue with ..." button. That's actually a more accurate meaning for that case than "sign in" is, even though the latter has a word in common.